inc/namespace.php

```diff
@@ -657,12 +657,12 @@ function output_ab_test_html_for_post( string $test_id, int $post_id, string $de
 	<ab-test
 		test-id="<?php echo esc_attr( $test_id ); ?>"
 		post-id="<?php echo esc_attr( $post_id ); ?>"
-		traffic-percentage="<?php echo get_ab_test_traffic_percentage_for_post( $test_id, $post_id ); ?>"
+		traffic-percentage="<?php echo esc_attr( get_ab_test_traffic_percentage_for_post( $test_id, $post_id ) ); ?>"
 		goal="<?php echo esc_attr( $test['goal'] ); ?>"
 		fallback="<?php echo esc_attr( $default_output ); ?>"
 		variants="<?php echo esc_attr( wp_json_encode( $variant_output, JSON_UNESCAPED_UNICODE | JSON_UNESCAPED_SLASHES ) ); ?>"
 	>
-		<?php echo $default_output; ?>
+		<?php echo $default_output; // phpcs:ignore ?>
```

Comment:
Because I had it handy:

```suggestion
		<?php echo $default_output; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>
```